code/iaas/model/src/main/java/io/cattle/platform/core/dao/InstanceDao.java

```diff
@@ -19,4 +20,6 @@
      */
     List<? extends Service> findServicesFor(Instance instance);
 
+    List<? extends Instance> listNonRemovedInstances(Account account, boolean forService);
```

Comment:
Is there somewhere else that makes this call passing in forService=true? (subsequent PR?)